src/lib/components/Accordion.react.js

```diff
@@ -28,6 +29,7 @@ Accordion.defaultProps = {
      * Any default prop values, e.g.
      *   congratulatoryMessage: 'You are amazing'
      */
+    showToggleText: true,
```

Comment:
As above
Accordion.defaultProps = {
    showToggleText: true,
};

Accordion.propTypes = {
    id: PropTypes.string,
    setProps: PropTypes.func,
    accordionHeadings: PropTypes.arrayOf(PropTypes.string),
    children: PropTypes.node,
    defaultSectionsOpen: PropTypes.arrayOf(PropTypes.bool),
    showToggleText: PropTypes.bool,
};

Reply:
This bit we already have the types and docstrings for:)